version: '3.9'
services:
  db:
    image: postgres:16.2-alpine
    container_name: throw-new-error-db
    restart: unless-stopped
    tty: true
    ports:
      - '${POSTGRES_PORT}:5432'
    volumes:
      - /var/lib/postgresql/data
      - ./docker/config/postgresql/postgresql.conf:/var/lib/postgresql/data/postgresql.conf
    environment:
      POSTGRES_DB: ${POSTGRES_DB}
      POSTGRES_USER: ${POSTGRES_USER}
      POSTGRES_PASSWORD: ${POSTGRES_PASSWORD}
      PGDATA: ${PGDATA}
